gui/freeadmin/static/lib/js/freeadmin.js

```diff
@@ -1416,6 +1416,19 @@ require([
 
     }
 
+    systemDatasetMigration = function() {
+        var dialog = new Dialog({
+            title: gettext("Warning!"),
+            id: "Warning_box_dialog",
+            content: domConstruct.create(
+                "p", {
+                    innerHTML: gettext(
+                        gettext("Warning: The action will result in migration of dataset.<br />Some services will be restarted.<br /><br />")
```

Comment:
innerHTML: gettext("The action will result in migration of dataset.") + "<br />" + gettext("Some services will be restarted.")